_posts/2016-04-10-docker-inside-docker-and-jenkins-2.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "Docker inside docker and overview about Jenkins 2"
+date:   2016-04-1 10:08:27
+categories: docker
+img: /img/docker.png
+tags: docker, oss, jenkins
+summary: "A little overview about Jenkins 2 but the main topic of the article
+is about how run docker inside docker to start a continuous integration system
+inside a container"
+priority: 0.6
+changefreq: yearly
+---
+
+Docker inside docker and Jenkins 2 Jenkins  is one of the most famous
+continuous integration and deployment tools, it’s written in Java and it helps
+you to manage your pipeline and all tasks that help you to put your code in
+production or manage your build.
+
+I am happy to announce that the version 2, in
```

Comment:
The announcement of Jenkins release of `version 2` few days ago, is one of the best release of this year in my opinion.